src/pages/HistoryPage/EpochForm.tsx

```diff
@@ -351,11 +351,13 @@ const EpochForm = ({
             data.repeat === 'weekly' ? 1 : data.repeat === 'monthly' ? 2 : 0,
           ...(data.description !== null && { description: data.description }),
         })
-      : createEpoch({
+      : /* createEpoch({
```

Comment:
in the future, for the sake of shipping more quickly, it's worth considering just creating a new API action, shipping it, then shipping the frontend change to use it, then going back and removing the old one (& optionally renaming the new action)

Reply:
@levity If incremental E2E QA testing for this wasn't a priority, I would have gone that route